assets/js/modules/analytics/index.js

```diff
@@ -61,21 +61,20 @@ export const registerModule = ( modules ) => {
 	} );
 };
 
+const isAnalyticsActive = ( select ) => ! isGA4DashboardView( select );
+const isAnalytics4Active = ( select ) => isGA4DashboardView( select );
```

Comment:
Maybe these should be defined somewhere common? It's used in a few files, but in some only once. 🤷🏻‍♂️

Reply:
I was mostly thinking a utility function where they were shared, but given it's all going away not too long from now, this is actually probably fine. I'm clearly overthinking it, let's 🚢